### ISSUE TITLE
[BUG] ReferenceError: options is not defined in src/.eleventy.js

### ISSUE THREAD

**Issue by sudkumar** [ampproject/eleventy-plugin-amp#21]
**Bug Description**

In a recent release of `0.2.0`, we introduced `filter` option to only process matched files. But accidentally, the `.eleventy.js` was also modified which caused this bug.

```diff
module.exports = {
-  configFunction: (eleventyConfig, options) => {
+  configFunction: (eleventyConfig, providedOptions) => {
    addAmpTransform(eleventyConfig, options);
``` 

As we can see the `options` is replaced with `providedOptions`  in the function arguments but not in the calls to other plugin functions. 

**Possible Solution**

Simply reverting the change will fix the issue i.e.

```diff
module.exports = {
-  configFunction: (eleventyConfig, providedOptions) => {
+  configFunction: (eleventyConfig, options) => {
    addAmpTransform(eleventyConfig, options);
```

**Comment by sudkumar** [comment]
PR #22 

@sebastianbenz  Please have a look.

**Comment by sebastianbenz** [comment]
This has been published in 0.2.1 - thanks again for fixing!

**Comment by sudkumar** [comment]
Thank you for this awesome plugin. ❤️
